src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -71,4 +89,95 @@ public enum JsonTokenType : byte
+        public void WriteStartObject(System.ReadOnlySpan<char> propertyName, bool suppressEscaping = false) { }
+        public void WriteStartObject(string propertyName, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<byte> propertyName, System.DateTime value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<byte> propertyName, System.DateTimeOffset value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<byte> propertyName, System.Guid value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<byte> propertyName, System.ReadOnlySpan<byte> value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<byte> propertyName, System.ReadOnlySpan<char> value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<byte> propertyName, string value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<char> propertyName, System.DateTime value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<char> propertyName, System.DateTimeOffset value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<char> propertyName, System.Guid value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<char> propertyName, System.ReadOnlySpan<byte> value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<char> propertyName, System.ReadOnlySpan<char> value, bool suppressEscaping = false) { }
+        public void WriteString(System.ReadOnlySpan<char> propertyName, string value, bool suppressEscaping = false) { }
+        public void WriteString(string propertyName, System.DateTime value, bool suppressEscaping = false) { }
+        public void WriteString(string propertyName, System.DateTimeOffset value, bool suppressEscaping = false) { }
+        public void WriteString(string propertyName, System.Guid value, bool suppressEscaping = false) { }
+        public void WriteString(string propertyName, System.ReadOnlySpan<byte> value, bool suppressEscaping = false) { }
+        public void WriteString(string propertyName, System.ReadOnlySpan<char> value, bool suppressEscaping = false) { }
+        public void WriteString(string propertyName, string value, bool suppressEscaping = false) { }
+        public void WriteStringValue(System.DateTime value) { }
+        public void WriteStringValue(System.DateTimeOffset value) { }
+        public void WriteStringValue(System.Guid value) { }
+        public void WriteStringValue(System.ReadOnlySpan<byte> utf8Text, bool suppressEscaping = false) { }
+        public void WriteStringValue(System.ReadOnlySpan<char> utf16Text, bool suppressEscaping = false) { }
+        public void WriteStringValue(string utf16Text, bool suppressEscaping = false) { }
```

Comment:
As another example of that, here for `WriteStringValue` we have `utf8Text` and `utf16Text`, but above in `WriteString` the parameters with the same types and that mean the same thing are just called `value`.  Why?